redaxo/src/addons/be_style/lib/command/compile.php

```diff
@@ -18,6 +18,9 @@ protected function configure()
             ->setDescription('Converts Backend SCSS files to CSS');
     }
 
+    /**
+     * @return void
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Eigentlich ist der Return-Type laut Elternmethode `int|null` in sf4, wobei null als return-Wert deprecated ist, und in sf5 ist der Return-Type `int`.

Bisher haben wir in Kindmethoden keine PHPDocs platziert. Ist das wirklich notwendig?

Reply:
> Bisher haben wir in Kindmethoden keine PHPDocs platziert. Ist das wirklich notwendig?

die typen sind durch das psalm-tool (psalter) generiert.
wenn ich sie manuell pflegen müsste würde ich sie nicht hinmachen, aber durch das tool und die damit gewonnen typesicherheit fand ich es charmant.

Reply:
> Eigentlich ist der Return-Type laut Elternmethode `int|null` in sf4, wobei null als return-Wert deprecated ist, und in sf5 ist der Return-Type `int`.

kann ich aktuell nicht beurteilen, muss ich im detail anschauen

Reply:
> die typen sind durch das psalm-tool (psalter) generiert.
wenn ich sie manuell pflegen müsste würde ich sie nicht hinmachen, aber durch das tool und die damit gewonnen typesicherheit fand ich es charmant.

Ich versuche inzwischen ja eigentlich eher, die PHPDocs zu minimieren, also dort wegzulassen, wo sie keine neue Information enthalten. Insbesondere
- Wenn der Typ schon als echter Typehint notiert ist
- Oder eben auch, wenn es schon an der Elternmethode notiert ist

Ich persönlich würde somit eher ein Tool nutzen, was sie in den Fällen entfernt (für den ersten Fall macht es ja der CS-Fixer) ;) 
Aber ist auch kein total wichtiger Punkt für mich.

Mir scheint es aber auch so, dass das Tool nicht überall bei Kindmethoden die Typen hinzugefügt hat. Ich weiß also gar nicht so genau, wieso es bei den Command-Klassen passiert ist.
Vielleicht wegen der Typinkonsistenz?